test/fuzz/BUILD

```diff
@@ -38,6 +38,22 @@ cc_binary(
     ],
 )
 
+cc_test(
+    name = "fuzz_doc_symbols",
+    srcs = [
+        "fuzz_doc_symbols.cc",
+    ],
+    tags = ["manual"],
+    visibility = ["//tools:__pkg__"],
+    deps = [
+        "//core",
+        "//main/lsp",
+        "//main/pipeline",
+        "//payload",
```

Comment:
We don't need the Ruby standard library, so there's no need for a payload.

You can use `//payload/binary:empty` and `//payload/text:empty` instead of `//payload`, then set `opts.noStdlib`. Then, initialization should be much faster.